.gitignore

```diff
@@ -4,3 +4,5 @@ venv
 __pycache__
 quail.conf.yaml*
 *.db
+/examples/hcvprod_pull/DANGER
```

Comment:
will this be read as an absolute path? maybe we should just disinclude anything with DANGER?

Reply:
I think maybe I was vague. It was just supposed to be an example.  I wrote this in step 3 " Goto $ cd examples/hcvprod_pull/DANGER folder. You may have to make this folder structure." telling them they may have to make these folders, as i did, to get the example to work.